core/client/src/main/java/alluxio/client/block/UnderStoreBlockInStream.java

```diff
@@ -38,10 +35,7 @@
  * storage client.
  */
 @NotThreadSafe
-public final class UnderStoreBlockInStream extends BlockInStream implements PositionedReadable {
-  private static final boolean PACKET_STREAMING_ENABLED =
-      Configuration.getBoolean(PropertyKey.USER_PACKET_STREAMING_ENABLED);
-
+public final class UnderStoreBlockInStream extends BlockInStream {
```

Comment:
what is the difference now between `UnderStoreBlockInStream` and `UnderFileSystemBlockInStream`?